tools/ci/test_runner.sh

```diff
@@ -66,15 +66,40 @@ else
 fi
 
 
+function get_public_master_url()
+{
+    local cluster_description_file="$(mktemp)"
+    local attempts=5
+    local master_ip
+    # We retry, since sometimes the cluster is created, but dcos-launch has intermittent problems describing it.
+    for attempt in $(seq 1 ${attempts}); do
+        # Careful to not use a pipeline!
+        if dcos-launch describe --info-path="${REPO_ROOT_DIR}/cluster_info.json" > "${cluster_description_file}" &&
+           master_ip=$(jq --raw-output --exit-status '.masters[0].public_ip' < "${cluster_description_file}")
+        then
+            echo "https://${master_ip}"
+            return 0
+        else
+            echo "parsing output of [dcos-launch describe] failed (attempt ${attempt})" >&2
+            local delay_sec=$((attempt*10))
```

Comment:
I have a feeling this will evaluate only once. Please declare it on top, along with the rest
```sh
local delay_sec
```
and only assign value on line 84